README.md

```diff
@@ -2,7 +2,11 @@
 
 This script is designed to allow simpler Docker containers for Clojure apps in dev and production using Immutant 2.
 
-It should be the "CMD" in your Clojure app's Dockerfile.
+It should be the "CMD" in your Clojure app's Dockerfile, like this:
+
+```
+CMD ["/bin/deploy-or-run"]
```

Comment:
Since the `CMD` is already set in _this_ Dockerfile, the child Dockerfile doesn't need it.